zebra-network/src/peer/connection.rs

```diff
@@ -91,82 +84,191 @@ impl Handler {
                 }
             }
             (Handler::Peers, Message::Addr(addrs)) => Handler::Finished(Ok(Response::Peers(addrs))),
+            // `zcashd` returns requested transactions in a single batch of messages.
```

Comment:
```suggestion
            // `zcashd` returns requested transactions in an uninterrupted batch of messages.
```

Reply:
I don't know if this is an accurate description of zcashd's behaviour. zcashd queues the messages in a single batch, but I don't know if it allows other types of messages to preempt that batch.